study-builder/tenants/testboston/tenant.yaml

```diff
@@ -1,12 +1,28 @@
 tenant:
   default_directory: Username-Password-Authentication
 
+databases:
+  - name: study-admin-authentication
+    strategy: auth0
+    enabled_clients:
+      - Auth0 Management API
```

Comment:
Idea is to NOT enable this database for any client, so that clients don't add users here. However, you need to enable database for AT LEAST one client, otherwise Auth0 will mark it as inactive and you can't event manually add users to it. Auth0 Management client is an arbitrary choice, but seems fine for now.